paasta_tools/setup_marathon_job.py

```diff
@@ -306,6 +306,8 @@ def log_bounce_action(line, level='debug'):
             level='event',
         )
         return None
+    elif (not all_old_tasks) and new_app_running:
+        return None
     else:
         return 60
```

Comment:
Yep, I think that would be sensible. Although the current diff will fix the issue so I'm happy if we just ship that too.